CITATION.bib

```diff
@@ -3,8 +3,8 @@ @misc{qNetVO
 	title   = {qNetVO},
 	howpublished =  {\url{https://github.com/ChitambarLab/qNetVO}},
 	url     = {https://github.com/ChitambarLab/qNetVO},
-	version = {v0.1.1},
+	version = {v0.1.2},
 	year    = {2022},
 	month   = {March},
-	doi     = {https://doi.org/10.5281/zenodo.6345835},
+	doi     = {https://doi.org/10.5281/zenodo.6345834},
```

Comment:
@trbromley this will point to the latest version automatically. Otherwise, it would be a pain to tag the version, then have to update the DOI after in a separate commit.

Reply:
if someone really cares to cite a particular version, then its easy for them to copy-paste the correct doi from zenodo, but I wager that no one actually cares about that

Reply:
Nice, cool that it automatically points to the latest!